server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1255,9 +1265,54 @@ public void deleteConsumer(
                 returnCerts.add(cert);
             }
         }
+        // we want to insert the content access cert to this list
+        if (consumer.getContentAccessCert() != null) {
+            try {
+                returnCerts.add(contentAccessCertService.getCertificate(consumer));
+            }
+            catch (IOException ioe) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+            catch (GeneralSecurityException gse) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+        }
         return returnCerts;
     }
 
+    @ApiOperation(notes = "Retrieves the body of the Content Access Certificate for the Consumer",
+        value = "getContentAccessBody")
+    @ApiResponses({ @ApiResponse(code = 404, message = "") })
+    @GET
+    @Path("{consumer_uuid}/content")
+    @Produces(MediaType.APPLICATION_JSON)
+    public ContentAccessListing getContentAccessBody(
+        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
+        @QueryParam("if-modified-since") Date since) {
```

Comment:
Regarding RFC 2616 as the format, I'd be inclined to agree if we were specifying the date in a header a la RFC 2616, but since we're exposing as a query param, I lean more towards making it consistent w/ other dates (esp. dates in responses). I would not make a fuss if we went with the RFC 2616 timestamp though. :smile: